tools/system_libs.py

```diff
@@ -467,6 +467,8 @@ def create_wasm_compiler_rt(libname):
                  'ashldi3.c', 'fixdfdi.c', 'floatdidf.c', 'lshrdi3.c', 'moddi3.c',
                  'trunctfdf2.c', 'trunctfsf2.c', 'umoddi3.c', 'fixunsdfdi.c', 'muldi3.c',
                  'divdi3.c', 'divmoddi4.c', 'udivdi3.c', 'udivmoddi4.c'])
+    files += files_in_path(path_components=['system', 'lib', 'compiler-rt'],
```

Comment:
Maybe move+rename `extras.c` to `system/lib/compiler-rt/lib/wasm_extras.c`

Reply:
Yeah.. I thought about giving it a different name.  Perhaps moving it up one level to its not in the compiler-rt source directory at all.

The thing is we have libc/extra.c so I thought it would be good to be consistent with that.